gemrb/GUIScripts/pst/NewLife.py

```diff
@@ -63,17 +63,12 @@ def OpenLUStatsWindow(Type = 1):
 
 	LevelUp = Type
 	if LevelUp:
-		import GUICommonWindows
 		import GUIREC
-		GUICommonWindows.OptionsWindow.SetVisible (False)
-		GUICommonWindows.PortraitWindow.SetVisible (False)
-		GUICommonWindows.ActionsWindow.SetVisible (False)
-		GUIREC.RecordsWindow.SetVisible (False)
 		# only TNO gets the main stat boosts
 		pc = GemRB.GameGetSelectedPCSingle ()
 		Specific = GemRB.GetPlayerStat (pc, IE_SPECIFIC)
 		if Specific != 2:
-			GUIREC.OpenLevelUpWindow ()
+			GUIREC.UpdateRecordsWindow(GUIREC.RecordsWindow)
```

Comment:
this is useless. `OpenLUStatsWindow` is called from `InitRecordsWindow` which is part of the `CreateTopWinLoader` that will also call `UpdateRecordsWindow`

as an additional comment that has nothing to do with your changes, but maybe you feel like addressing anyway:

I hate that `OpenLUStatsWindow` does things other than open the window. I wish that other stuff would be moved to the singular location where `OpenLUStatsWindow` is called.

Reply:
That's not a problem, I can split it out - I just don't usually think too much about changing stuff unless it isn't working, everyone else is way more qualified than me to make design decisions :)